src/commands/Misc/reddituser.ts

```diff
@@ -0,0 +1,212 @@
+import { CommandStore, KlasaMessage, Timestamp } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { fetch, roundNumber, getColor, cutText } from '../../lib/util/util';
+import { Reddit } from '../../lib/types/definitions/Reddit';
+import { MessageEmbed } from 'discord.js';
+import { BrandingColors } from '../../lib/util/constants';
+
+const COMPLEXITY_LEVELS = ['very low', 'low', 'medium', 'high', 'very high', 'very high'];
+
+export default class extends SkyraCommand {
+
+	private joinedRedditTimestamp = new Timestamp('MMMM d YYYY');
```

Comment:
Is there any reason for this to be a class property?

Reply:
Consistency with other pieces.